README.md

```diff
@@ -78,6 +78,10 @@ Available options:
         The body of the request. See '-b/body' for more details.
   -H/--headers K=V
         The request headers.
+  --har FILE
+        When provided, Autocannon will use requests from the HAR file. 
+        CAUTION: you have to to specify one (or more) domain using URL option: only the HAR requests to the same domains will be considered.
```

Comment:
Does this throw? Is the domain overridden?

Reply:
Neither of the two.
if you specify URL https://A.com, requests from the HAR file to https://B.com will simply be skipped.
Please note that we support multiple domains:
- assuming HAR contains requests to A, B and C,
- assuming you specify A and B urls,
- the first connection will fire requests to A, 
- the next connection will fire requests to B,
- requests to C will be ignored.

Overriding would not make sense, as you're likely to issue a request to an unsupported path or with unsupported parameters.

We can throw as well, but supporting the case above will be harder.

Reply:
I don't really understand this behavior and it looks like it's quite confusing. Let's just throw for now and then we leave this to further iterations? (Open an issue after landing)

Reply:
It would be very hard to use.
Tools that generate HAR like browsers usually wraps requests from many domains.
People will have to alter their files accordingly to trim out requests they don't want.

Just to be sure about your intention:
Given user can pass several urls from the CLI, do you want to throw an error if the HAR file contains a requests that does not target one of these urls?

Reply:
> It would be very hard to use. Tools that generate HAR like browsers usually wraps requests from many domains.
People will have to alter their files accordingly to trim out requests they don't want.

I understand now! Maybe we can print a warning about the domains that are ignored?